pallets/crowdloan-rewards/Cargo.toml

```diff
@@ -23,9 +23,13 @@ sp-io = { default-features = false, git = 'https://github.com/paritytech/substra
 sp-runtime = { default-features = false, git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.4' }
 sp-std = { default-features = false, git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.4' }
 
+cumulus-pallet-parachain-system = { default-features = false, git = 'https://github.com/paritytech/cumulus.git', branch = 'polkadot-v0.9.4' }
+cumulus-primitives-core = { default-features = false, git = 'https://github.com/paritytech/cumulus.git', branch = 'polkadot-v0.9.4' }
+cumulus-test-relay-sproof-builder = { git = 'https://github.com/paritytech/cumulus.git', branch = 'polkadot-v0.9.4' }
```

Comment:
move it to dev-dependencies if don't actually use it

Reply:
Remove std and move packages to dev-dependencies
https://github.com/polkafoundry/newpolka/pull/55/commits/4321a8e57e11f400e1c8f5f6a9ef91ce2bcead11#diff-272c2905c8bbd7c1fb16473920231b37d1d0a8912563f784f1e2405cb2f786fbR30